tests/base_test.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from pages.home import HomePage
+
+
+class BaseTest:
+
+    def login_new_user(self, mozwebqa, user):
```

Comment:
You're right, I'm not keen on lots of UI interactions happening in a fixture. I'm also not keen on base test classes though. I'm okay with leaving this as it is for now, but after consideration I think this is best placed in a base page object as individual functions that can be chained together in a helper when we don't need to include checks.

Something like:

```python
def test_that_needs_new_user_logged_in(new_user):
    page = HomePage()
    page.go_to_page()
    page.login_and_configure_initial_profile(new_user)
    ...

def test_configure_initial_profile(new_user):
    home_page = HomePage()
    home_page.go_to_page()
    edit_profile_page = page.login(new_user)
    assert edit_profile_page.is_the_current_page
    assert edit_profile_page.is_user_logged_in
    edit_profile_page.type_name(user['name'])
    edit_profile_page.type_username(user['name'])
    assert edit_profile_page.is_privacy_policy_checkbox_checked
    home_page = edit_profile_page.click_save_button('home_page')
    return home_page.is_the_current_page
```

Anyway, I'm okay with leaving this as is.